### ISSUE TITLE
`/debug` mode no longer works in Console

### ISSUE THREAD

**Issue by andytudhope** [status-im/status-react#3677]
### User Story

As a 3rd party developer, I want to turn `/debug` mode on in `Console` so that I can build and test my DApp in Status.

### Description

*Type*: Bug

*Summary*:  You need to enable `debug` mode in Status in order to for `status-dev-cli` to work between your machine and device. It no longer does.

#### Expected behavior
[comment]: # Sending the `/debug` command to `Console` after enabling `Developer mode` should result in a message being passed back to me about how `status-dev-cli scan` should now return the IP of my connected device.

#### Actual behavior
[comment]: # The `/debug` message is seen, but there is no response.

### Reproduction

- Open Status
- Go to profile and turn on Developer Mode in Advanced Options.
- Navigate to `Console`
- Send `/debug` message

### Additional Information
* Status version: Latest nightlies, but also on 0.9.15 in Play Store, where I can't even access `Advanced Options`.
* Operating System: Android

#### Logs

[comment]: # (if on Android please replicate bug whilst running adb logcat)
```
s",["debug",50],"preview"]
03-26 12:08:49.279 10378 10434 D StatusModule: startCallJail
03-26 12:08:49.284 10378 10434 D StatusModule: endCallJail
03-26 12:08:49.786  2617  2617 I InputContext: input context reset success: true error: 0 input_state_id 3553
03-26 12:08:49.786  2617  2617 I LatinMotionHandler: consumeEvent() : GestureTypingEnabled = true
03-26 12:08:56.735  3159  3159 I GeofencerStateMachine: sendTransitions: location=Location[fused -26.099244,28.054341 acc=23 et=+1d1h23m38s209ms alt=1677.4000244140625 {Bundle[mParcelledData.dataSize=640]}]
03-26 12:08:56.754  2039  2061 I ActivityManager: Killing 10218:com.android.vending:instant_app_installer/u0a19 (adj 906): empty #17
03-26 12:08:56.775  3159  8449 W ctxmgr  : [AclManager]No 2 for (accnt=account#164240226#, com.google.android.gms(10011):UserLocationProducer, vrsn=12221000, 0, 3pPkg = null ,  3pMdlId = null ,  pid = 3159). Was: 2 for 1, account#164240226#
03-26 12:08:56.778  3159  8449 W ctxmgr  : [AclManager]No 2 for (accnt=account#-404528431#, com.google.android.gms(10011):UserLocationProducer, vrsn=12221000, 0, 3pPkg = null ,  3pMdlId = null ,  pid = 3159). Was: 2 for 1, account#-404528431#
03-26 12:08:56.785  3159  8449 W ctxmgr  : [AclManager]No 2 for (accnt=account#1164180932#, com.google.android.gms(10011):UserLocationProducer, vrsn=12221000, 0, 3pPkg = null ,  3pMdlId = null ,  pid = 3159). Was: 2 for 1, account#1164180932#
03-26 12:08:56.822  2039 25725 D ActivityManager: cleanUpApplicationRecord -- 10218
03-26 12:09:02.579 10378 10510 D JavaScriptCore: WARN [03-26|10:09:02] message loop                             peer=482484b9198530ee err=EO
03-26 12:09:02.704 10378 10510 D JavaScriptCore: WARN [03-26|10:09:02] message loop                             peer=d40871fc3e11b264 err=EO
03-26 12:09:03.499 10378 10510 D JavaScriptCore: WARN [03-26|10:09:03] message loop                             peer=86ebc843aa51669e err=EO
03-26 12:09:13.015 10378 10388 I art     : Background sticky concurrent mark sweep GC freed 187880(5MB) AllocSpace objects, 2(88KB) LOS objects, 34% free, 25MB/39MB, paused 704us total 139.726ms
03-26 12:09:15.346 11881 11881 W sfq     : Already started: sfq@ce2166a
03-26 12:09:15.564 11881 12129 E wua     : Job(ItemMessageAttachmentsCacheEvictorJob#0.33) Enqueued job 'ItemMessageAttachmentsCacheEvictorJob' failed
03-26 12:09:15.564 11881 12129 E wua     : java.lang.IllegalArgumentException: Attachment cache cannot be null.
03-26 12:09:15.564 11881 12129 E wua     : 	at oyk.a(Unknown Source)
03-26 12:09:15.564 11881 12129 E wua     : 	at ozy.a(Unknown Source)
03-26 12:09:15.564 11881 12129 E wua     : 	at xth.a(SourceFile:2)
03-26 12:09:15.564 11881 12129 E wua     : 	at wtm.b(SourceFile:2)
03-26 12:09:15.564 11881 12129 E wua     : 	at wtn.a(Unknown Source)
03-26 12:09:15.564 11881 12129 E wua     : 	at xth.a(SourceFile:2)
03-26 12:09:15.564 11881 12129 E wua     : 	at xso.run(SourceFile:1)
03-26 12:09:15.564 11881 12129 E wua     : 	at wko.run(Unknown Source)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
03-26 12:09:15.564 11881 12129 E wua     : 	at java.lang.Thread.run(Thread.java:761)

```

**Comment by andytudhope** [comment]
Already known: https://github.com/status-im/status-react/issues/3440

**Comment by nabgilby** [comment]
in the latest status version on the Android store [ 0.9.19 (6034)] and latest released nightlies [0.9.20 (6058]) /debug no longer works, again it seems.

**Comment by nabgilby** [comment]
debug is still broken on 0.9.20 (6065)??
